santosfamilyfoundation/RuinaMain
# [RUIN -303] Save photo to export files

# Description

The ability to add a crash diagram was recently added to the app. However, the app was not able to export the image. Now the photo will be exported with the report. For HTML and PDF files, the image will be placed in the file under the crash diagram. For Excel files, the photo is not able to save in the file but will be saved with the report.

### Note
Tried to use `excelJS` library that allowed images to be added to the workbook. However, there were so many errors that I could not figure out. Ultimately decided to stick with current library, `xlsx`, and export the photo as a separate file. This is not the most ideal scenario. Please leave a comment with suggestions.

# Testing

1. Open a report
2. Add a crash photo
3. Export report
4. Click "Save to Local Device"
5. Choose .html
6. Save Report
7. Open downloaded report
8. Check that report has image
9. Return to app
10. Go back to Final Report screen
11. Click "Save to Local Device"
12. Choose .xlsx
13. Save Report
14. Open downloaded report
15. Check download folder for crash photo
